ci/pipelines/perf-tests.yml

```diff
@@ -68,30 +68,485 @@ resources:
     source:
       url: https://hooks.slack.com/services/((slack-notification-secret))
 
+  - name: paas
+    type: cf-cli
+    icon: cloud-upload-outline
+    source:
+      api: https://api.cloud.service.gov.uk
+      org: govuk-pay
+      space: build
+      username: ((cf-username))
+      password: ((cf-password))
+      cf_cli_version: 7
+
+  - name: every-night-at-11pm
+    icon: alarm
+    source:
+      start: "23:00"
+      stop: "23:30"
+      location: Europe/London
+    type: time
+
 resource_types:
   - name: slack-notification
     type: docker-image
     source:
       repository: cfcommunity/slack-notification-resource
       tag: latest
 
+  - name: cf-cli
+    type: registry-image
+    source:
+      repository: nulldriver/cf-cli-resource
+
 groups:
-  - name: perf-tests
+  - name: complete-perf-tests
+    jobs:
+      - scale-and-run-all-simulations
+  - name: individual-perf-tests
+    jobs:
+      - run-payment-simulation-perf-test
+      - run-search-payment-simulation-perf-test
+      - run-self-service-simulation-perf-test
+  - name: environment-scaling
     jobs:
-      - run-perf-test
+      - scale-up-databases
+      - scale-up-services
+      - scale-up-all
+      - scale-down-databases
+      - scale-down-services
+      - scale-down-all
   - name: perf-tests-build
     jobs:
       - build-and-push-perf-tests
   - name: update-perf-tests-pipeline
     jobs:
       - update-perf-tests-pipeline
+  - name: all-jobs
```

Comment:
What's the purpose of this "all-jobs"?

Reply:
Just to see all the things in the serial group together. To be honest I didn't mean to leave it in here :)

Reply:
I'll remove it in one of the follow on